test/test_pullback_tensors.jl

```diff
@@ -127,6 +127,5 @@ end
  
 @testset "variational equation" begin
     include("define_vector_fields.jl")
-    @test CG_tensor(var_rot_double_gyre, [0.5,0.5],  [0.0, 5.0], 0.0, tolerance= 1.e-4) isa SymmetricTensor
+    @test (@inferred CG_tensor(var_rot_double_gyre, (0.5, 0.5), [0.0, 5.0], 0.0)) isa SymmetricTensor
```

Comment:
FYI, I changed here from the vector `[0.5, 0.5]` to the tuple `(0.5, 0.5)`, because this makes it type stable, and is the recommended way of representing coordinates of spatial points, also in our documentation, I think.